site-scrapers/TuftsMC/index.js

```diff
@@ -0,0 +1,135 @@
+const https = require("https");
+const file = require("../../lib/file");
+const fetch = require("node-fetch");
+const { site, requestInit } = require("./config");
+
+module.exports = async function GetAvailableAppointments(
+    browser,
+    pageService = defaultPageService()
+) {
+    console.log(`${site.name} starting.`);
+    const webData = await ScrapeWebsiteData(browser, pageService);
+    console.log(`${site.name} done.`);
+    return {
+        ...site,
+        ...webData,
+        timestamp: new Date(),
+    };
+};
+
+function defaultPageService() {
+    return {
+        async fetchAvailability() {
+            return fetchAvailability();
+        },
+    };
+}
+
+async function ScrapeWebsiteData(browser, fetchService) {
+    // Initialize results to no availability
+    const results = {
+        availability: {},
+        hasAvailability: false,
+    };
+
+    /* Note:
+        site has next button if 'button[data-test="desktop-controls-right-arrow"]'
+        is not disabled
+    */
+
+    const response = await fetchService.fetchAvailability();
+
+    results.availability = {
+        ...results.availability,
+        ...parseResponse(response),
+    };
+    // }
```

Comment:
mind removing this?

Reply:
What?! Remove my single attempt at an emoji? It records my wife and daughter's comment: "you need a haircut!" Seriously, I will remove it.